packages/fleather/lib/src/rendering/editor.dart

```diff
@@ -608,7 +608,8 @@ class RenderEditor extends RenderEditableContainerBox
     // as well.
     if (selectionChanged ||
         (cause == SelectionChangedCause.longPress ||
-            cause == SelectionChangedCause.keyboard)) {
```

Comment:
I think it's better to extract this into an extension method on `SelectionChangedCause.longPress`. WDYT? @amantoux

Reply:
@Amir-P what do you mean?

Reply:
Something like this:
```
Extension on SelectionChangedCause {
  bool get shouldSendEvent => this == SelectionChangedCause.longPress || ...
}
```
WDYT? @amantoux

Reply:
Ok, I'm not sure it will be needed though it the future